tst/teststandard/bugfix.tst

```diff
@@ -1083,7 +1083,7 @@ gap> StructureDescription(testG(8,2));
 gap> StructureDescription(testG(8,3));
 "C3 x QD16"
 gap> StructureDescription(testG(8,4));
-"((C16 x C2) : C2) : C2"
+"((C16 : C2) : C2) : C2"
```

Comment:
This will not work with no packages. The proper test should check for ````StructureDescription(testG(8,4):nice);```` 
That would be stable with or without packages.